string group
---
float32[] xyz
